code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -52,11 +34,7 @@
 		COLOR_PINK = "Pink",
 		COLOR_MAROON = "Brown"
 	)
-	var/static/list/bottle_styles = list("bottle", "small_bottle", "wide_bottle", "round_bottle", "reagent_bottle")
-	var/list/safe_chem_list = list("antihol", "charcoal", "epinephrine", "insulin", "teporone", "silver_sulfadiazine", "salbutamol",
-									"omnizine", "stimulants", "synaptizine", "potass_iodide", "oculine", "mannitol", "styptic_powder",
-									"spaceacillin", "salglu_solution", "sal_acid", "cryoxadone", "blood", "synthflesh", "hydrocodone",
-									"mitocholide", "rezadone", "menthol", "diphenhydramine", "ephedrine", "iron", "sanguine_reagent")
+	var/list/datum/chemical_production_mode/production_modes = list()
```

Comment:
Can you make this an associated list instead, and use custom ids on each mode? (This would also prevent duplicates in any weird cases.) I used a new variable here, but you could just as easily use font awesome icon name in `production_icon`
```
var/datum/chemical_production_mode/new_mode = new /datum/chemical_production_mode/pills()
production_modes[new_mode.mode_name] = new_mode 
new_mode = new /datum/chemical_production_mode/patches()
production_modes[new_mode.mode_name] = new_mode
new_mode = new /datum/chemical_production_mode/bottles()
production_modes[new_mode.mode_name] = new_mode